lib/travis/build/script/node_js.rb

```diff
@@ -263,6 +231,19 @@ def prepend_path(path)
           def yarn_req_not_met
             "$(travis_vers2int $(echo `node --version` | tr -d 'v')) -lt $(travis_vers2int #{YARN_REQUIRED_NODE_VERSION})"
           end
+
+          def is_win?
+            config[:os].downcase.strip == 'windows'
+          end
+
+          def setup_os
+            if is_win?
+              sh.echo "Using NVS for managing Node.js versions on Windows (BETA)", ansi: :yellow
+              sh.export 'NVS_HOME', '$ProgramData/nvs', echo: false
+              sh.cmd 'git clone --single-branch -b joshk-msys_nt https://github.com/joshk/nvs $NVS_HOME'
```

Comment:
Can whatever changes are in the `joshk/nvs` fork be moved/integrated into a `travis-ci/nvs` fork, assuming they can't be accepted upstream yet?

Reply:
https://github.com/jasongin/nvs/pull/106 has been merged, so we can change this to the upstream.